### PR TITLE
Putting examples in boxes, adding missing verification steps

### PR DESCRIPTION
**Pull request by MichalMaler**
Signed-off-by: Michal Maléř <mmaler@redhat.com>

> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by MichalMaler** [comment]
@tolusha Hello Anatolii, please, can you provide the OC step for the second procedure:
https://www.eclipse.org/che/docs/che-7/end-user-guide/creating-a-workspace-from-remote-devfile/#overriding-devfile-value[…]ng-factory-parameters_che ?

It could be filled in this file:
https://github.com/eclipse/che-docs/blob/1a679d76fde335dced8a667e8605d5d5d3e89888/[…]ls/proc_overriding-devfile-values-using-factory-parameters.adoc , I prepared a box for it already.

File: modules/end-user-guide/partials/proc_overriding-devfile-values-using-factory-parameters.adoc [hunk 1]
```diff
@@ -153,3 +154,8 @@ attributes:
 projects:
 ...
 ----
+====
+
+.Verification steps
+
+. Using {prod-short} Dashboard, move to the *Devfile* tab of the newly created workspace and check its values.
```
Comment (sleshchenko) [inline modules/end-user-guide/partials/proc_overriding-devfile-values-using-factory-parameters.adoc]:
```suggestion
. Using {prod-short} Dashboard, move to the *Devfile* tab of the newly created workspace and check its content.
```

**Review by sleshchenko — APPROVED** [review]
`modules/end-user-guide/partials/proc_overriding-devfile-values-using-factory-parameters.adoc` LGTM

**Review by yhontyk — APPROVED** [review]
LGTM, please rebase and feel free to merge whenever

**Pull request merged**